packages/manager/apps/container/package.json

```diff
@@ -32,6 +32,7 @@
     "@ovh-ux/manager-config": "^7.5.0",
     "@ovh-ux/manager-core-api": "^0.8.0",
     "@ovh-ux/manager-react-components": "^1.39.1",
+    "@ovh-ux/manager-react-shell-client": "^0.7.3",
```

Comment:
This is architecturally not correct to include this depenedency.
Can you add a TODO to avoid it?

Reply:
Sure Anoop 👍